src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/CommentsIndentation/context/case4/Test.java

```diff
@@ -0,0 +1,12 @@
+package TreeWalker.CommentsIndentation;
+
+public interface Test {
+int a = 1;
+    // violation without filter
+}
+
+enum Test2 {
```

Comment:
Same as https://github.com/checkstyle/patch-filters/pull/321#discussion_r481483475
Please add a comment above this enum.

Reply:
done